kubectl_build/Tiltfile

```diff
@@ -47,3 +53,34 @@ def kubectl_build(ref, context, build_args={}, dockerfile='Dockerfile', live_upd
 
     custom_build(ref, command, [context], disable_push=True, skips_local_docker=True, live_update=live_update,
                  match_in_env_vars=match_in_env_vars, ignore=ignore, entrypoint=entrypoint)
+
+
+def image_build(*args, **kwargs):
```

Comment:
hmmmm...it seems weird to have a function alias?

are you aware that starlark has a way to import functions under a different name?

```
load('ext://kubectl_build', image_build='kubectl_build')
```

Reply:
yes I do, but I tried

~~~python
load('ext://kubectl_build', image_build='kubectl_build')
~~~

and

~~~python
load('ext://kubectl_build', 'image_build')
~~~

and liked the latter better in my Tiltfiles.

If you prefer to not have it, I can remove it.

Reply:
:+1: that's fine! I wanted to make sure you knew the feature existed (since not everyone does)